source/_integrations/dynalite.markdown

```diff
@@ -89,6 +91,16 @@ area:
           description: Name of the area.
           required: true
           type: string
+        template:
+          description: "Type of template to use for the area. Supported values are: 'room', 'trigger', and 'timecover'. They are described in details below in the **template** section. If the template parameters are different than defaults, they can be overriden in this section as well"
+          require: false
+          type: string
+          default: No template
+        TEMPLATE_PARAMS:
+          description: "This can be any of the settings of the template. For example, for template 'room': 'room_on' and 'room_off' are possible options"
```

Comment:
```suggestion
          description: "This can be any of the settings of the template. For example, for template `room`: `room_on` and `room_off` are possible options."
```